nixos/modules/programs/plotinus.nix

```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.plotinus;
+in
+{
+  ###### interface
+
+  options = {
+    programs.plotinus = {
+      enable = mkOption {
+        default = false;
+        description = ''
+          Whether to enable the Plotinus GTK+3 plugin.  Plotinus provides a
+          popup (triggered by Ctrl-Shift-P) to search the menus of a
+          compatible application.
+        '';
+        type = types.bool;
+      };
+    };
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+    environment.variables.GSETTINGS_SCHEMA_DIR = "${pkgs.plotinus}/share/gsettings-schemas/plotinus-0.2.0/glib-2.0/schemas";
```

Comment:
And here, `XDG_DATA_DIRS` is probably better since `GSETTINGS_SCHEMA_DIR` cannot be used more than once.

Reply:
That sounds like a good idea.  But I don't know how to use XDG_DATA_DIRS in this context as it gives me an error that it is specified by another module.

Reply:
Oh, got ya - using a list fixes it.

Reply:
`environment.variables.XDG_DATA_DIRS = [ "${pkgs.plotinus}/share/gsettings-schemas/plotinus-0.2.0" ];` should work. What module is that?